app.py

```diff
@@ -10,7 +10,7 @@
 import sys
 import yaml
 
-app = Flask(__name__)
+app = Flask(__name__, static_url_path='')
```

Comment:
I would suggest to add `static_url_path='/static'` instead. If not specified, the custom assets won't be loaded from `/data`.